src/memory-analyzer/analyze_symbol.cpp

```diff
@@ -212,6 +298,24 @@ exprt gdb_value_extractort::get_non_char_pointer_value(
   }
 }
 
+bool gdb_value_extractort::points_to_member(
+  const pointer_valuet &pointer_value) const
+{
+  if(pointer_value.pointee.find("+") != std::string::npos)
```

Comment:
Did you mean `std::find` or something internal? I use it twice so I added a wrapper method into `pointer_valuet` with a nice name.

Reply:
Right here I was thinking about `string::find`, but I could be wrong about the other cases. With `string::find` I'd just (when possible) actually use single characters rather than one-character strings.